File: fern/snippets/authentication/typescript/auth-config.ts
import { Composio } from '@composio/core';

const composio = new Composio();

const authConfig = await composio.authConfigs.create('PERPLEXITYAI', {
  name: 'Perplexity AI',
  type: 'use_custom_auth',
  credentials: {},
  authScheme: 'API_KEY',
});

console.log(authConfig);

File: fern/snippets/authentication/python/auth-config.py
from composio import Composio

composio = Composio()

# Use custom auth
auth_config = composio.auth_configs.create(
    toolkit="perplexityai",
    options={
        "type": "use_custom_auth",
        "auth_scheme": "API_KEY",
        "credentials": {}
    },
)
print(auth_config)
